### ISSUE TITLE
[feature request] Move the prompt text to the top, maybe rephrase it

### ISSUE THREAD

**Issue by mac-chaffee** [duosecurity/duo_openvpn_as#24]
## Detailed Description
When you log in, the prompt text that appears seems kind of confusing to me:

https://github.com/duosecurity/duo_openvpn_as/blob/4f8f5a627112b8c1fe515ef55faf56b9f04f3c2d/duo_openvpn_as.py#L606

It's not immediately apparent to me that I could/should type "push" in the box. The prompt that my org uses is "Enter 'push', 'sms', 'phone', or a Duo app passcode".

## Use Case
I think moving the prompt variable to the top of the file along with the rest of the config vars would make it clearer that admins can edit it. Even if we rephrase the prompt to mention all those commands, maybe some orgs have "sms" disabled and would want to remove that part of it. Or maybe admins would want to include a link to internal docs or a helpdesk link.

## Workarounds
Users can manually edit line 606 of the script.

**Comment by AaronAtDuo** [comment]
@mac-chaffee How does https://github.com/duosecurity/duo_openvpn_as/pull/25 look to you?  Is that what you had in mind?

**Comment by mac-chaffee** [comment]
Looks good, thanks!
